frontend/public/components/overview/build-overview.tsx

```diff
@@ -14,13 +16,14 @@ import { ResourceLink, SidebarSectionHeading, useAccessReview } from '../utils';
 const conjugateBuildPhase = (phase: BuildPhase): string => {
   switch (phase) {
     case BuildPhase.Cancelled:
-      return 'was cancelled';
+      return i18next.t('overview~was cancelled');
```

Comment:
These messages next to take a parameter with the build name.

Reply:
Could you provide more details ? Thanks.

Reply:
The message needs to be something like `Build {{buildName}} was cancelled`. Concatenating string fragments into sentences causes problems during translation.

It will require some rework to the code unfortunately.

Reply:
In the original English message, it includes <BuildNumberLink build={build} /> which is a link ,as below.
After reworking like the format  'Build {{buildName}} was cancelled', <BuildNumberLink build={build} /> will be removed.
I am not sure if it is ok to remove <BuildNumberLink build={build} />.
Please sugges it.

----------------
The original English message code:
 const statusTitle = (
    <div>
      Build <BuildNumberLink build={build} /> {conjugateBuildPhase(phase)}
      {lastUpdated && (
        <>
          {' '}
          <span className="build-overview__item-time text-muted">({fromNow(lastUpdated)})</span>
        </>
      )}
    </div>
  );
------------

Reply:
Could you let me know what are the steps to find the GUI for build-overview.tsx  in Openshift console? Then I could check if the update is correct  on my local Openshift console .

Reply:
Finished reworking the codes.